doc/source/conf.py

```diff
@@ -66,7 +66,7 @@
 # Intersphinx mapping
 intersphinx_mapping = {
     "python": ("https://docs.python.org/dev", None),
-    "ansys-dpf-core": ("https://dpf.docs.pyansys.com", None),
+    "ansys-dpf-core": ("https://dpf.docs.pyansys.com/release/0.7", None),
```

Comment:
With the next release of the multi-version docs action, this should get a URL that always points to the latest stable release. 
For now, we need to use the specific URL for the released page.